deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: review
+title:  "Functional Map of the World"
+tags:   deep-learning, classification, bounding-boxes, CNN, LSTM, remote-sensing, dataset
+author: Charles Authier
+pdf:  https://arxiv.org/pdf/1711.07846
+cite:
+  authors: "Gordon Christie, Neil Fendley, James Wilson, Ryan Mukherjee"
+  title:   "Functional Map of the World"
+  venue:   "CVPR 2018"
+---
+
+## Dataset
+Over 1 million images from over 200 countries. Each image, provide at least one bounding box annotation containing one of 63 categories.
```

Comment:
For each image, the goal is to provide? 
or 
There is at least one bounding box associated to one of the 63 possible categories for each image

Reply:
There is at least one bounding box associated to one of the 63 possible categories for each image.

Reply:
well you should change the sentence above :)